kotlin-ktor/src/main/java/com/baeldung/graphql/server/api/ConferenceSubscription.kt

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.graphql.server.api
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.server.operations.Subscription
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+class ConferenceSubscription : Subscription {
+
+    @GraphQLDescription("Emits single, newly created conferences")
+    fun conference(): Flow<Int> = ConferencePublisher.conferenceFlow.map { it.id!! }
```

Comment:
Should we rename this to `conferenceIds` instead, to match the actual behaviour?

Reply:
Renamed to `conferenceId`